inc/saiswitch.h

```diff
@@ -558,11 +558,10 @@ typedef enum _sai_switch_attr_t
      * #SAI_NATIVE_HASH_FIELD_ETHERTYPE\]
      * #SAI_HASH_ATTR_UDF_GROUP_LIST empty list
      *
-     * The object id is read only, while the object attributes can be modified
      *
      * @type sai_object_id_t
      * @objects SAI_OBJECT_TYPE_HASH
-     * @flags READ_ONLY
+     * @flags CREATE_AND_SET
```

Comment:
hmm, could this attribute be create and set, and allow null, which null will mean 'use default' and it should be well defined, this way we will avoid more "default" objects created on the switch since i just noticed that i will also need to add those fields to "hidden" objects and during syncd restart i will have to "skip" those hash creations which are just readonly and it will require special logic